src/system_window.cpp

```diff
@@ -788,4 +814,82 @@ void SystemWindow::UpdateBrightness()
 	}
 }
 
-} // namespace PanzerChasm
+bool SystemWindow::ScreenShot( const std::filesystem::path& file ) const
```

Comment:
It is not a good idea for `SystemWindow` class to save screenshots.
Leave in this class only necessary functionality - framebuffer grabbing and move other screenshots-making functionality into separate function, unrelated to `SystemWindow` class.

Reply:
I've separated the file manipulation functions to save_load.hpp now and renamed ScreenShot to CaptureScreen.
I've also added ControlsMenu getInSetMode exception to support remap of screenshot key binding at the ControlsMenu.